core/scripts/chaincli/handler/debug.go

```diff
@@ -229,73 +236,75 @@ func (k *Keeper) Debug(ctx context.Context, args []string) {
 	if checkResult.UpkeepFailureReason != 0 {
 		message(fmt.Sprintf("checkUpkeep failed with UpkeepFailureReason %d", checkResult.UpkeepFailureReason))
 	}
+
 	if checkResult.UpkeepFailureReason == uint8(encoding.UpkeepFailureReasonTargetCheckReverted) {
-		// TODO use the new streams lookup lib
-		//mc := &models.MercuryCredentials{k.cfg.MercuryLegacyURL, k.cfg.MercuryURL, k.cfg.MercuryID, k.cfg.MercuryKey}
-		//mercuryConfig := evm.NewMercuryConfig(mc, core.StreamsCompatibleABI)
-		//lggr, _ := logger.NewLogger()
-		//blockSub := &blockSubscriber{k.client}
-		//_ = streams.NewStreamsLookup(packer, mercuryConfig, blockSub, keeperRegistry21, k.rpcClient, lggr)
+		mc := &models.MercuryCredentials{k.cfg.MercuryLegacyURL, k.cfg.MercuryURL, k.cfg.MercuryID, k.cfg.MercuryKey}
+		mercuryConfig := evm.NewMercuryConfig(mc, core.StreamsCompatibleABI)
+		lggr, _ := logger.NewLogger()
+		blockSub := &blockSubscriber{k.client}
+		streams := streams.NewStreamsLookup(packer, mercuryConfig, blockSub, k.rpcClient, keeperRegistry21, lggr)
 
 		streamsLookupErr, err := packer.DecodeStreamsLookupRequest(checkResult.PerformData)
 		if err == nil {
 			message("upkeep reverted with StreamsLookup")
 			message(fmt.Sprintf("StreamsLookup data: {FeedParamKey: %s, Feeds: %v, TimeParamKey: %s, Time: %d, ExtraData: %s}", streamsLookupErr.FeedParamKey, streamsLookupErr.Feeds, streamsLookupErr.TimeParamKey, streamsLookupErr.Time.Uint64(), hexutil.Encode(streamsLookupErr.ExtraData)))
-			if streamsLookupErr.FeedParamKey == feedIdHex && streamsLookupErr.TimeParamKey == blockNumber {
-				message("using mercury lookup v0.2")
```

Comment:
I think we need to keep this because the "allowed to use mercury" check is only necessary for v0.2 mercury lookups. Also I think it is nice for debugging if we continue to log the link "using mercury lookup v0.X" so the dev can match that against expectations